### DISCUSSION TITLE
Collaboration?

### DISCUSSION THREAD

**Discussion by ctag-fh-kiel** [copych/AcidBox#34] [General]
Hi, I'm the author of CTAG-TBD: https://github.com/ctag-fh-kiel/ctag-tbd
Some videos on it: https://www.youtube.com/playlist?list=PLB5iCbhcvJ2qdD7s1o9wsvQ9qtsCUWVLR

Would you be interested in collaboration, e.g. dsp-code, sequencer apps etc?

Cheers

**Comment by copych** [comment]
Hello @ctag-fh-kiel thank you for the invitation! I checked the videos provided, and I must admit that you've already done a long way. Your CTAG TBD is a great flexible product already. I usually gladly participate in any music/sound connected projects, and I would say "yes, sure, I will", but I want to be sober and measure my abilities adequately. I'm just a hobbyist, not a skilled coder, I just play with affordable ESP32 series, either I don't have a strong math basis to deal freely with DSP stuff, and I dedicate my time to the hobby randomly. Also I already participate in a kind of collaboration here https://github.com/copych/AciduinoBox , and have a couple of stalled unfinished projects which I'd like to build in firmware in the end. If someday your invitation is still valid, I'd join. For now I can offer the code from my repo's which is public and mostly MIT-licensed. Feel free to contact if you want to discuss some ideas or have some questions about ESP32, if I know the answer, I'd gladly share it with you. Cheers

**Reply by ctag-fh-kiel** [reply]
Hey there, sounds good, thanks for checking out TBD, be in touch :)
